src/DataProtection/DataProtection/src/LoggingExtensions.cs

```diff
@@ -255,4 +255,25 @@ private static bool IsLogLevelEnabledCore([NotNullWhen(true)] ILogger? logger, L
 
     [LoggerMessage(73, LogLevel.Debug, "Key {KeyId:B} method {MethodName} failed. Retrying.", EventName = "RetryingMethodOfKeyAfterFailure")]
     public static partial void RetryingMethodOfKeyAfterFailure(this ILogger logger, Guid keyId, string methodName, Exception exception);
+	
+    [LoggerMessage(74, LogLevel.Information, "Deleting file '{FileName}'.", EventName = "DeletingFile")]
+    public static partial void DeletingFile(this ILogger logger, string fileName);
+
+    [LoggerMessage(75, LogLevel.Information, "Failed to delete file '{FileName}'.  Not attempting further deletions.", EventName = "FailedToDeleteFile")]
+    public static partial void FailedToDeleteFile(this ILogger logger, string fileName, Exception exception);
```

Comment:
Why is this log information-level while `FailedToRemoveDataFromRegistryKeyValue` is error-level? Should it be error-level for both?

Reply:
Looks like a copy-paste error though, in practice, neither level will actually be printed unless `--verbose` is passed.